android/app/src/main/java/com/ylitse/MainApplication.java

```diff
@@ -28,7 +25,6 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
-          packages.add(new RNFirebaseMessagingPackage()); // FCM
```

Comment:
Miks kaikki Firebase jutut poistettu?

Reply:
https://rnfirebase.io/migrating-to-v6#removing-the-react-native-firebase-packages